docs/README.md

```diff
@@ -73,7 +73,7 @@ To add another hook use `husky add`.
 For example:
 
 ```shell
-npx husky add .husky/commit-msg 'npx --no-install commitlint --edit "$1"'
+npx husky add .husky/commit-msg 'npx --no commitlint -- --edit "$1"'
```

Comment:
```suggestion
npx husky add .husky/commit-msg 'npx --no commitlint --edit "$1"'
```
It seems unnecessary with npx
https://docs.npmjs.com/cli/v8/commands/npx#examples